src/mlp.c

```diff
@@ -56,7 +56,7 @@ static OPUS_INLINE float tansig_approx(float x)
        x=-x;
        sign=-1;
     }
-    i = (int)floor(.5f+25*x);
+    i = (int)(.5f+25*x);
```

Comment:
Why this change? Does it speed things up?

Reply:
Because floor() is defined with `double floor(double x);`, it is causing 2 type casting and a function call in quite heavy loop.

It's speedup is <0.1% in encoding process though.